integration/compression_test.go

```diff
@@ -0,0 +1,87 @@
+//go:build integration
+// +build integration
+
+package integration
+
+import (
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/bitrise-io/go-utils/v2/command"
+	"github.com/bitrise-io/go-utils/v2/env"
+	"github.com/bitrise-io/go-utils/v2/log"
+	"github.com/bitrise-steplib/steps-save-cache/compression"
+)
+
+func Test_compression(t *testing.T) {
```

Comment:
Maybe we should add a check that the `ztsd` option is available to this, with a helpful warning if it's not.

Reply:
Good idea, done!